Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture.py

```diff
@@ -116,14 +118,35 @@ def build_iterator(self, sub_feed, indicator_type):
         try:
             response.raise_for_status()
         except requests.exceptions.HTTPError:
-            return_error('{} - exception in request: {} {}'.format(self.SOURCE_NAME, response.status_code, response.content))
+            if "Insufficient credits" in response.text:
+                return_error("'Insufficient Credits' error was returned from Recorded Future. \n"
+                             "Try increasing the integration's fetch interval in order to decrease the amount of API"
+                             " requests from Recorded Future. ")
+            else:
+                return_error('{} - exception in request: {} {}'.format(self.SOURCE_NAME, response.status_code, response.content))
 
         data = response.text.split('\n')
 
         csvreader = csv.DictReader(data)
 
         return csvreader
 
+    def calculate_indicator_score(self, risk_from_feed):
+        """Calculates the Dbot score of an indicator based on its Risk value from the feed.
+        Args:
+            risk_from_feed (str): The indicator's risk value from the feed
+        Returns:
+            int. The indicator's Dbot score
+        """
+        dbot_score = 0
+        risk_from_feed = int(risk_from_feed)
+        if risk_from_feed >= self.threshold or risk_from_feed >= 65:
+            dbot_score = 3
+        elif risk_from_feed >= 5:
```

Comment:
what does 5 mean?

Reply:
![image](https://user-images.githubusercontent.com/50294648/73137214-77d87780-405e-11ea-90e5-eab661c58958.png)
this should explain the scoring.
after consulting @idovandijk , we decided to treat malicious+very malicious as malicious,
unusual+suspicious as suspicious